src/patternfly/components/Card/docs/code.md

```diff
@@ -12,5 +12,6 @@ A card is a generic rectangular container that can be used to build other compon
 | `.pf-c-card__footer` | `<div>` | Creates the footer of a card. |
 | `.pf-c-card__head` | `<div>` | Creates the head of the card where images or actions can go. |
 | `.pf-c-card__actions` | `<div>` | Creates an actions element to be used in the card head. |
+| `.pf-m-compact` | `.pf-c-card` | Creates a compact variation of the card component, that involves smaller font sizes and spacing. |
 | `.pf-m-no-fill` | `.pf-c-card__body` | Sets a `.pf-c-card__body` to not fill the available space in `.pf-c-card`. `.pf-m-no-fill` can be added to multiple card bodies. |
 | `.pf-m-hoverable` | `.pf-c-card` | Modifies the card to include hover styles on `:hover` |
```

Comment:
Noticed the needs a period.
```suggestion
| `.pf-m-hoverable` | `.pf-c-card` | Modifies the card to include hover styles on `:hover`. |
```

Reply:
I'll add my grammar nit; there should not be a comma before "that" :-)